app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -156,11 +156,19 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
         let offSetDate = new Date();
         offSetDate.setDate(offSetDate.getDate() - offSet);
         offSetDate.setHours(0, 0, 0);
-        const currStartDate = filter.startDate ? new Date(filter.startDate) : offSetDate;
+        // if no date offset/ start date this means the user is looking for all content
+        // unix epoch used as a default start date for all content
+        const currStartDate = filter.startDate
+          ? new Date(filter.startDate)
+          : filter.dateOffset !== undefined
+          ? offSetDate
+          : new Date(1970, 0, 1);
         const prevStartDate = new Date(currStartDate.getTime() - 7 * dayInMs);
         const currEndDate = filter.endDate
           ? new Date(filter.endDate)
-          : new Date(currStartDate.getTime() + offSet * dayInMs - 1);
+          : filter.dateOffset !== undefined
+          ? new Date(currStartDate.getTime() + offSet * dayInMs - 1)
+          : new Date();
```

Comment:
End date should be today when selecting all content (aka no `filter.endDate` and `filter.dateOffset === undefined`

Reply:
This is still not correct.  But we'll merge it to fix the other issues.  If no date is selected then it doesn't need a start or end date.  Elasticsearch will simply return all found stories sorted by date and limited by 500. 
Change the date to now can result in the next time you click the search button possibly being stale and you would miss the most recent published stories.